projects/components/src/datagrid/filters/datagrid-string-filter.component.ts

```diff
@@ -53,6 +62,12 @@ export class DatagridStringFilterComponent extends DatagridFilter<string, Datagr
         return filterBuilder.equalTo(value).getString();
     }
 
+    protected onBeforeSetConfig(config: DatagridStringFilterConfig) {
+        this.placeholderValue = config?.placeholderValue
```

Comment:
This can be made more succinct with

 ```config?.placeholderValue || this.translationService.translateAsync('vcd.cc.datagrid.common.filter.items')```